pyoos/collectors/nerrs/nerrs_soap.py

```diff
@@ -30,6 +45,8 @@ def get_stations(self):
                 if val is None:
                     val = ""
                 s[child.tag] = val
+            s['Longitude'] = -float(s['Longitude'])
```

Comment:
> We need to find a way to test this PR against a working NERRS configuration.

I've been cleaning up a notebook that includes a pyoos nerrs example. I'll post it by the end of tomorrow, and will use pyoos master (if ready by then) to run it.

> One more reason to enforce negative longitudes instead of hoping they are always positive.

Again, go right ahead, no harm done! But I don't think NERRS is being sloppy per se (and therefore likely to change things suddenly); they chose to advertise their longitudes in a station "metadata" request as a positive value (West longitudes) b/c that kind of assumption is a little too common when people always work in one lat-lon quadrant. I doubt they'll change that on a whim, but heck, it doesn't hurt for us to be extra careful.